src/test/java/com/cloudbees/hudson/plugins/folder/FolderTest.java

```diff
@@ -278,6 +286,32 @@ private void copyFromGUI(Folder f, JenkinsRule.WebClient wc, String fromName, St
             }
         });
     }
+    
+    @Issue("JENKINS-32359")
+    @Test public void shouldProperlyPersistFolderPropertiesOnMultipleReloads() throws Exception {
+        Folder folder = r.jenkins.createProject(Folder.class, "myFolder");
+        r.jenkins.setAuthorizationStrategy(new ProjectMatrixAuthorizationStrategy());
+        
+        // We add a stub property to generate the persisted list
+        // After that we save and reload the config in order to drop PersistedListOwner according to the JENKINS-32359 scenario
+        folder.addProperty(new FolderCredentialsProvider.FolderCredentialsProperty(new DomainCredentials[0]));
```

Comment:
I need to save a non-empty properties list to the disk and reload it. It makes the code to bypass the <code>if (properties == null)</code> clause